docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md

```diff
@@ -20,36 +32,23 @@ Your bill is calculated based on your usage of the following dimensions:
 - Backup storage
 - Data transfer
 
-Instance vCPU capacity makes up the majority of your bill, and is the easiest to understand and control. It's purely a function of your total number of vCPUs used and how long they have been running. The cluster's per-hour charge includes free allowances for disk storage, backup storage, and data transfer. If you use more than the free allowance, you incur overages on top of the base vCPU capacity cost.
-
-| Dimension | Allowance/vCPU per month |
-|---|---|
-| Disk storage | 50 GB |
-| Backup storage | 100 GB |
-| Data transfer – Same Region | 1000 GB |
-| Data transfer – Cross Region (APAC) | 10 GB |
-| Data transfer – Cross Region (Other regions) | 10 GB |
-| Data transfer – Internet | 10 GB |
-
-You can see the approximate cost for your vCPUs when [creating](../../cloud-basics/create-clusters/) and [scaling](../../cloud-clusters/configure-clusters/) clusters, as shown in the following illustration. **+ Usage** in this context refers to any potential overages from exceeding the free allowances for disk storage, backup storage, and data transfer.
-
-![Cluster Settings Edit Infrastructure](/images/yb-cloud/cloud-clusters-settings-edit.png)
+The following calculations reflect the Standard plan rate card.
```

Comment:
@ddhodge can you please link to the pricing page from here ('Standard plan rate card')? Or add a link for more information, for example, 'For more information on plan based pricing, see <yugabyte.com/pricing>'